subprojects/testing-native/src/main/java/org/gradle/nativeplatform/test/xctest/tasks/XcTest.java

```diff
@@ -103,4 +108,23 @@ protected File getRunScript() {
         }
         return runScriptFile;
     }
+
+    /**
+     * Sets the test name patterns to be included in execution.
+     * Classes or method names are supported, wildcard '*' is not supported.
+     */
+    @Option(option = "tests", description = "Sets test class or method name to be included, '*' is not supported.")
+    public XcTest setTestNameIncludePatterns(List<String> testNamePattern) {
```

Comment:
should just have the option on the super method and get rid of this method and the same in `Test`.